cli/cmd/plugin/ui/web/tanzu-ui/src/shared/services/Provider.service.tsx

```diff
@@ -11,24 +11,29 @@ export interface ProviderInfo {
 export interface ProviderData {
     name: string;
     logo: string;
+    logoClass: string;
 }
 
 const providerInfo: ProviderInfo = {
     aws: {
         name: 'AWS',
         logo: AwsLogo,
+        logoClass: 'aws-logo-img',
```

Comment:
I like the way you did these logo classes; particularly: thanks for not concatenating the provider name with `-logo-img`, but having them be a first-class field